user/templates/templates.md

```diff
@@ -155,22 +155,24 @@ and also that it is not set as the default template.
 
 The procedure for uninstalling a template depends on how it was created.
 
-If the template was originaly created by cloning another template, then you can
+If the template was originally created by cloning another template, then you can
 delete it the same way as you would any other qube. In the Qube Manager,
 right-click on the template and select **Delete qube**. (If you're not sure,
 you can safely try this method first to see if it works.)
 
-If, on the other hand, the template came pre-installed or was installed by
-installing a template package in dom0, per the instructions
-[above](#installing), then you must execute the following type of command in
-dom0 in order to uninstall it:
+If, on the other hand, the template came pre-installed or was installed through
+a template package in dom0 using the instructions [above](#installing), then you
+must execute the following commands in dom0 in order to uninstall it.
 
+List the installed templates:
 ```
-$ sudo dnf remove qubes-template-<DISTRO_NAME>-<RELEASE_NUMBER>
+$ qvm-template list --installed | awk -F " " '{print $1}'
```

Comment:
```suggestion
$ qvm-template list --installed | awk 'NF==3 {print $1}'
```

the default field separator is a whitespace, so no need to add `-F " "`. In addition, `qvm-template list --installed` adds a `Template installed` at the end of its output when piping into a command (and without a pipe, it's the first line :thinking: ), so this could be hidden by displaying lines with 3 fields with `NF == 3`, or using an extra `grep -v ^Installed`.

Reply:
Thank you for confirming the moving "Template installed" title. This patch:
https://github.com/QubesOS/qubes-core-admin-client/pull/245
should resolve it allowing a simpler awk command. Your suggestions helped here.

Reply:
nice, it would even be better to not display the header if you use `--installed` or `--available` because that's not really useful in the output, but it seems to be more work than your fix.

With that change `| awk 'NR>1 { print $1 }'` should work to discard the first line.

Reply:
Added the pattern to skip outputting the redundant title. Thank you for the review.